### PR TITLE
Update paper.md

### PR DESCRIPTION
**Pull request by farhadrclass**
Changes based on Prof. Orban review

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -130,47 +129,49 @@ end
...
 
 ```julia
-using NLPModelsModifiers # define also LBFGSModel
+using NLPModelsModifiers # define also LSR1Model
 
 lsr1_LeNet = NLPModelsModifiers.LSR1Model(LeNetNLPModel)
-
 callback_lsr1 = 
-  (sr1_nlpmodel, solver, stats) -> FluxNLPModels.minibatch_next_train!(sr1_nlpmodel.model)
+  (lsr1_LeNet, solver, stats) -> FluxNLPModels.minibatch_next_train!(lsr1_LeNet)
```
Comment (paraynaud) [inline paper/paper.md:174]:
```suggestion
  (lsr1_LeNet, solver, stats) -> FluxNLPModels.minibatch_next_train!(
                                                 lsr1_LeNet.model
                                               )
```

File: paper/paper.md [hunk 2]
```diff
@@ -105,10 +105,9 @@ function get_MNIST(;T = Float32)
   return xtrain, ytrain, xtest, ytest
 end
 
-function getdata(; T = Float32) #T for types
+function get_data(; T = Float32)
   ENV["DATADEPS_ALWAYS_ACCEPT"] = "true"
-  # Loading Dataset	
-  xtrain, ytrain, xtest, ytest = get_MNIST(;T)
+  xtrain, ytrain, xtest, ytest = get_MNIST(;T=T)
```
Comment (paraynaud) [inline paper/paper.md:110]: For information, `xtrain, ytrain, xtest, ytest = get_MNIST(;T)` also works

File: paper/paper.md [hunk 3]
```diff
@@ -130,47 +129,49 @@ end
 train_loader, test_loader = create_batch()
 ```
 
-To transfer the LeNet model using the FluxNLPModel constructor, one needs to pass the model that was defined in Flux, the loss function, as well as the train and test data loaders.
+To construct the LeNet model as a FluxNLPModel, one needs to pass the model that was defined in Flux, the loss function, as well as the train and test data loaders.
```
Comment (paraynaud) [inline paper/paper.md:132]:
```suggestion
To cast the LeNet model as an FluxNLPModel, one needs to pass the model that was defined in Flux, the loss function, as well as the train and test data loaders.
```
This is what I read from the comments.

File: paper/paper.md [hunk 4]
```diff
@@ -130,47 +129,49 @@ end
...
 Flux.jl allows flexibility to define any loss function we need.
-In this example, we will use the built-in `Flux.logitcrossentropy` function as our loss function. 
+We will use the built-in `Flux.logitcrossentropy`.
+
 ```julia
 using FluxNLPModels
```
Comment (paraynaud) [inline paper/paper.md:138]:
```suggestion
```

File: paper/paper.md (hunk 1 above)
Comment (paraynaud) [reply paper/paper.md:174]: After some thoughts, there was no mistake with `sr1_nlpmodel`, it is just a variable name in an anonymous function definition.

File: paper/paper.md [hunk 5]
```diff
@@ -130,47 +129,49 @@ end
...
-Another choice to train `LeNetNLPModel` is the LBFGS linesearch solver of JSOSolvers.jl:
+Another choice to train `LeNetNLPModel` is the LBFGS linesearch solver:
 ```julia
 solver_stats = lbfgs(LeNetNLPModel; callback, max_time)
-
 test_accuracy = FluxNLPModels.accuracy(LeNetNLPModel)
 ```
 
-To enhance the `LeNetNLPModel`, an `LSR1` (or `LBFGS`) approximation of the Hessian can be employed and fed into the `trunk` solver. The trunk solver utilizes a quadratic trust-region method with a backtracking line search. To integrate the LSR1 approximation and trunk into the training process, the code can be modified as outlined below:
+To `LeNetNLPModel`, an `LSR1` approximation of the Hessian can be employed and fed into the `trunk` solver. The trunk solver utilizes a trust-region method with a backtracking line search. To integrate the LSR1 approximation and trunk into the training process, the code can be modified as:
```
Comment (paraynaud) [inline paper/paper.md:167]:
```suggestion
To exploit any non-convexity present in `LeNetNLPModel`, an `LSR1` approximation of the Hessian can be employed and fed into the `trunk` solver.
The trunk solver utilizes a trust-region method with a backtracking line search.
To integrate the LSR1 approximation and trunk into the training process, the code can be modified as:
```

**Review by paraynaud — CHANGES_REQUESTED** [review]
Thanks @farhadrclass, I made some suggestion when I saw differences with the remarks made in the .pdf

File: paper/paper.md (hunk 2 above)
Comment (farhadrclass) [reply paper/paper.md:110]: yes but I think Prof. Orban wants it to be clear, no?

Comment (paraynaud) [reply paper/paper.md:110]: You did well, my comment was just for information :)

File: paper/paper.md (hunk 3 above)
Comment (paraynaud) [reply paper/paper.md]: It makes sense to me, but I just applied the comment of the .pdf

**Pull request merged**